libcxx/test/std/numerics/c.math/hermite.pass.cpp

```diff
@@ -0,0 +1,310 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14
+
+#include <cassert>
+#include <cmath>
+#include <limits>
+#include <vector>
+#include <array>
```

Comment:
```suggestion
#include <array>
#include <cassert>
#include <cmath>
#include <limits>
#include <vector>
```

Reply:
Fixed in 840321277748f8d62682e6d92420e0b1ac39848e.